README.md

```diff
@@ -201,6 +201,30 @@ This will match the export name `"react-server"` and `"edge-light"` then use the
 
 `bunchee` has support for checking the package bundles are matched with package exports configuration.
 
+### Wildcard Exports (Experimental)
+
+You can use wildcard `"./*"` to match all [exportable entries](https://github.com/huozhi/bunchee?tab=readme-ov-file#exportable-entries).
+
+```json
+{
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.cjs"
+    },
+    "./*": {
+      "import": "./dist/*.mjs",
+      "require": "./dist/*.cjs"
+    }
+  }
+}
+```
+
+#### Exportable Entries
+
+- Files under the `src` directory, except the index file.
+- Directories with the index file.
+
```

Comment:
prefer to have a 1~2 lines description of this feature instead of the in-short bullet points, might make users who konws exports but not faimilar with wildcards to understand how it works

Reply:
### Wildcard Exports (Experimental)

You can use the wildcard `"./*"` to reduce repeating exports configuration. For example:


```json
{
  "exports": {
    ".": {
      "import": "./dist/index.mjs",
      "require": "./dist/index.cjs"
    },
    "./a": {
      "import": "./dist/a.mjs",
      "require": "./dist/a.cjs"
    },
    "./b": {
      "import": "./dist/b.mjs",
      "require": "./dist/b.cjs"
    },
    "./c": {
      "import": "./dist/c.mjs",
      "require": "./dist/c.cjs"
    }
  }
}
```

can be simplified to:

```json
{
  "exports": {
    ".": {
      "import": "./dist/index.mjs",
      "require": "./dist/index.cjs"
    },
    "./*": {
      "import": "./dist/*.mjs",
      "require": "./dist/*.cjs"
    }
  }
}
```

Reply:
How about providing a comparing example?

Reply:
👍 example is great, I think we should provide an example reversed, from defined wildcards, matching the existing project structure, and output the assets. It will work with nodejs resolving

Reply:
### Wildcard Exports (Experimental)

You can use the wildcard `"./*"` to reduce repeating the exports configuration.

For example:

```json
{
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"
    },
    "./*": {
      "import": "./dist/*.mjs",
      "require": "./dist/*.cjs"
    }
  }
}
```

The asterisk `*`  will be replaced with your entry files. Such as:

```
- my-lib/
  |- src/
    |- foo/
      |- index.ts
    |- bar.ts
    |- index.ts
  |- package.json
```

This will match the export names `"foo"` and `"bar"` and will be treated as:

```json
{
  "exports": {
    ".": {
      "types": "./dist/index.d.ts",
      "import": "./dist/index.js"
    },
    "./foo": {
      "import": "./dist/foo.mjs",
      "require": "./dist/foo.cjs"
    },
    "./bar": {
      "import": "./dist/bar.mjs",
      "require": "./dist/bar.cjs"
    }
  }
}
```

Reply:
Looks good, the last part of `"foo"` and `"bar"` could be simplified a bit, saying they're treated as new entries, and mentioning that they can be matched with the wildcard exports logic as it's nodejs feature

Reply:
@huozhi Edited the previous comment!

Reply:
perfect, lets give it a go